services/inc/services_dynalib.h

```diff
@@ -49,6 +49,7 @@ DYNALIB_FN(services,LED_RGB_SetChangeHandler)
 DYNALIB_FN(services, log_print_direct_)
 DYNALIB_FN(services, LED_GetColor)
 //DYNALIB_FN(services, log_level_active)
```

Comment:
The `log_level_active()` is something we'd tried adding but strangely caused the system to crash, so it was pulled (we had no time to investigate.)

Perhaps you could sketch out how we might allow log client code to determine if logging at a given level with a given category would produce any log output? This will allow client code to short circuit any expensive logging operations.